app/controllers/split_checkout_controller.rb

```diff
@@ -72,6 +74,13 @@ def remove_ship_address_errors
     @order.errors.delete("ship_address.zipcode")
   end
 
+  def replace_bill_address_phone_error
+    @order.errors["bill_address.phone"].each do |error|
+      @order.errors.add(:phone, error)
+    end
+    @order.errors.delete("bill_address.phone")
+  end
```

Comment:
Couldn't this be done in the translation file? Something like `bill_address/phone: Customer Phone`

Reply:
I agree, you're right, modified and squashed ;)
See commit: 9a03023b6b